src/opm/io/eclipse/ESmry.cpp

```diff
@@ -219,40 +232,81 @@ ESmry::ESmry(const std::string &filename, bool loadBaseRunData)
         }
 
         std::string smspecFile = std::get<0>(smryArray[n]);
-        std::string unsmryFile = smspecFile.substr(0, smspecFile.size() - 6) + "UNSMRY";
 
-        EclFile unsmry(unsmryFile);
-        unsmry.loadData();
-
-        std::vector<EclFile::EclEntry> list1 = unsmry.getList();
-
-        // 2 or 3 arrays pr time step.
-        //   If timestep is a report step:  MINISTEP, PARAMS and SEQHDR
-        //   else : MINISTEP and PARAMS
+        rootN = formattedVect[n] ? smspecFile.substr(0, smspecFile.size() - 7) : smspecFile.substr(0, smspecFile.size() - 6);
+        
+        // check if multiple or unified result files should be used 
```

Comment:
Second time this comment.

Reply:
keeping this, removed the comment in hpp file